File for remage

tdixon97 opened the pull request:
Add a small remage output file and the missing dt map for B99000A.
@gipert should we wait for a remage release so that this file corresponds to it? Maybe nice

tdixon97 committed:
update dt maps to include the BEGe

tdixon97 committed:
add test data for remage

tdixon97 committed:
add also the gdml

gipert commented:
i would like to discuss about what remage data to put here

fhagemann commented:
Maybe a quick comment from the Julia team:
We can read the remage output using `LegendHDF5IO.jl` without problems:

```julia
using LegendHDF5IO
stp = lh5open("data/remage/stp.lh5") do h
    LegendHDF5IO.readdata(h, "stp")
end
```
works like a charm :+1:

tdixon97 committed:
move atts to flattened data

tdixon97 commented:
I updated _reboost_ to place the attrs on the flattened data. I also updated the test file here https://github.com/legend-exp/legend-testdata/pull/33
I had a look and it seems to work (also reading into Julia)

gipert commented:
wait didn't we say that units should be attached to the outermost HDF5 object?

tdixon97 commented:
Then it cant be read by julia

gipert commented:
We should document these things in our specification... @oschulz?

oschulz commented:
> wait didn't we say that units should be attached to the outermost HDF5 object?

No, definitely innermost. Anything else would also be weird implementation-wise. After all, the vectors-of-vectors is just a nested view of a flat vector - can't just convert/attach units when doing a view. :-)

gipert commented:
Ok, but we should add this to our spec docs

gipert commented:
Superseded by #39.
